firebase-app-distribution/ktx/api.txt

```diff
@@ -0,0 +1,10 @@
+// Signature format: 2.0
+package com.google.firebase.app.distribution.ktx {
+
+  public final class FirebaseAppDistributionKt {
+    ctor public FirebaseAppDistributionKt();
```

Comment:
nit: formatting on the file seems to be off. get rid of the blank space at the start of line 6 and line 7